### ISSUE TITLE
different duration in the paper and the code

### ISSUE THREAD

**Issue by zhangyunzhen2027** [xw1216/EEG-FM-Bench#13]
Thanks for releasing EEG-FM-Bench — it's been very helpful for reproducing evaluations.

While aligning our preprocessing with the official code, we noticed an inconsistency between **Table 1 (Duration)** in the paper and the **actual `wnd_div_sec` used for finetune** in the repo.

### Observation

For **Workload**, Table 1 reports:

> Workload | Mental Stress | 19 | **10** | 1537 | 300 | 297 | Binary Classification

But in [`data/dataset/workload.py`](https://github.com/xw1216/EEG-FM-Bench/blob/main/data/dataset/workload.py):

```python
wnd_div_sec: int = 10  # class default / pretrain

BUILDER_CONFIGS = [
    BUILDER_CONFIG_CLASS(name='pretrain'),
    BUILDER_CONFIG_CLASS(name='finetune', is_finetune=True, wnd_div_sec=4),
]
```

So downstream evaluation appears to use **4-second** windows, not 10-second ones.

This pattern also appears for other datasets where finetune overrides the default window length, e.g.:

| Dataset | Table 1 Duration | Code pretrain | Code finetune |
|---------|------------------|---------------|---------------|
| Workload | 10 | 10 | **4** |
| SEED | 10 | 10 | **4** |
| PhysioMI | 4 | 12 | **4** |
| TUAB | 30 | 30 | **15** |
| Most others | matches | same | same |

Recomputing Workload with non-overlapping 4s windows gives a total of **2134** samples (`1537+300+297` in the paper; locally we get the same total with a slightly different subject split: `1535/297/302`), which also suggests the reported sample counts are based on **4s**, not 10s.

### Questions

1. For Table 1, does **Duration** refer to the **pretrain** window, the **finetune** window, or something else (e.g., a nominal trial length)?
2. For Workload (and similarly SEED / PhysioMI / TUAB), what is the intended rationale for using a **shorter finetune window** than the config default / pretrain setting?
3. For fair reproduction, should we treat the **finetune `wnd_div_sec` in code** as the ground truth for evaluation sample construction?

Happy to open a PR to clarify this in the README / table caption if that would be useful.

Thanks!
